src/test/java/org/ays/endpoints/InstitutionEndpoints.java

```diff
@@ -2,15 +2,7 @@
 
 import io.restassured.response.Response;
 import lombok.experimental.UtilityClass;
-import org.ays.payload.AdminsListPayload;
-import org.ays.payload.ApplicationRegistration;
-import org.ays.payload.Assignment;
-import org.ays.payload.RejectReason;
-import org.ays.payload.RequestBodyAssignments;
-import org.ays.payload.RequestBodyInstitution;
-import org.ays.payload.RequestBodyUsers;
-import org.ays.payload.User;
-import org.ays.payload.UserCredentials;
+import org.ays.payload.*;
```

Comment:
https://www.baeldung.com/intellij-disable-wildcard-import
```suggestion
import org.ays.payload.AdminsListPayload;
import org.ays.payload.ApplicationRegistration;
import org.ays.payload.Assignment;
import org.ays.payload.RejectReason;
import org.ays.payload.RequestBodyAssignments;
import org.ays.payload.RequestBodyInstitution;
import org.ays.payload.RequestBodyUsers;
import org.ays.payload.User;
import org.ays.payload.UserCredentials;
```